### ISSUE TITLE
Error: `wxt submit init` - missing required config

### ISSUE THREAD

**Issue by GODrums** [wxt-dev/wxt#400]
### Describe the bug

Executing the `wxt submit init`-command gives an `ERROR: Missing required config`. I've tried both having no `.env.submit`-file as well as an empty one in the repository.

Log:
```bash
> pnpm wxt submit init
ℹ Initialize or update an existing .env.submit file.

 ERROR  Missing required config 

  at resolveConfig (node_modules\.pnpm\publish-browser-extension@2.1.0\node_modules\publish-browser-extension\dist\cli.cjs:3404:11)
  at init (node_modules\.pnpm\publish-browser-extension@2.1.0\node_modules\publish-browser-extension\dist\cli.cjs:3517:26)
  at CAC.<anonymous> (node_modules\.pnpm\publish-browser-extension@2.1.0\node_modules\publish-browser-extension\dist\cli.cjs:3905:11)
  at CAC.runMatchedCommand (node_modules\.pnpm\cac@6.7.14\node_modules\cac\dist\index.js:614:34)
  at CAC.parse (node_modules\.pnpm\cac@6.7.14\node_modules\cac\dist\index.js:541:12)
  at Object.<anonymous> (node_modules\.pnpm\publish-browser-extension@2.1.0\node_modules\publish-browser-extension\dist\cli.cjs:3911:5)
  at Module._compile (node:internal/modules/cjs/loader:1376:14)
  at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
  at Module.load (node:internal/modules/cjs/loader:1207:32)
  at Module._load (node:internal/modules/cjs/loader:1023:12)

```

### To Reproduce

This happened in one of my [projects](https://github.com/GODrums/BetterBuff). Steps to reproduce the bug in my project:
```
> git clone https://github.com/GODrums/BetterBuff.git
> pnpm install
> pnpm wxt submit init
```

### Expected behavior

Following the [documentation](https://wxt.dev/guide/publishing.html), ideally the command would invoke promts to help generating a new `.env.submit`-file.

### Environment

```
  System:
    OS: Windows 11 10.0.22631
    CPU: (16) x64 AMD Ryzen 7 7800X3D 8-Core Processor
    Memory: 17.99 GB / 31.21 GB
  Binaries:
    Node: 20.10.0 - C:\Program Files\nodejs\node.EXE
    npm: 10.2.5 - C:\Program Files\nodejs\npm.CMD
    pnpm: 8.14.1 - ~\AppData\Local\pnpm\pnpm.CMD
    bun: 1.0.18 - ~\.bun\bin\bun.EXE
  Browsers:
    Edge: Chromium (121.0.2277.83)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    vite: ^5.0.12 => 5.0.12
    wxt: ^0.16.0 => 0.16.0
```

### Additional context

Tested it with Windows 11 as well as WSL2, but the error stays the same for both systems.

**Comment by Timeraa** [comment]
Can reproduce! +1

**Comment by aklinker1** [comment]
Yeah, stupid me was doing the testing with a fully setup .env.submit file, forgetting that obviously people setting up the file for the first time will be missing config 🤦

I'm traveling for the next few days, but will fix it ASAP. Code change needs to be here if anyone else wants to look into fixing it:

https://github.com/aklinker1/publish-browser-extension/blob/99cd58336397c88a33e636a40fb3b241a07bd2da/src/init.ts#L16

We need to break up resolving the options from the environment from validating the config. Only need to validate for the actual submit command, init we just need to grab environment variables.

**Comment by aklinker1** [comment]
Fixed by https://github.com/aklinker1/publish-browser-extension/commit/89bfe5d320202566c4b7073986ab773f1f2c2698

**Comment by aklinker1** [comment]
Closed by https://github.com/wxt-dev/wxt/commit/9318346ab3e584b71556e001bc9dd66125607bf8, released in [`v0.16.1`](https://github.com/wxt-dev/wxt/releases/tag/v0.16.1)
